app.py

```diff
@@ -43,6 +42,15 @@
     "https://api.snapcraft.io/api/v1/snaps/details/{snap_name}"
     "?channel=stable"
 )
+details_query_headers = {
+    'X-Ubuntu-Series': '16',
+    'X-Ubuntu-Architecture': 'amd64',
```

Comment:
I think you need to look at the installs across all the architectures. Even if we're not intending on exposing the different architectures in the page, we should at least count them in the graph (i.e. loop over all of them)

Reply:
The details API is not the same as the metrics API. The metrics query isn't filtered by architecture. The details API just gets data about a specific revision, not metrics. Also, AFAIK (check with @cprov) the details API will always choose an architecture - if I don't specify it it will default to `amd64`.